index.html

```diff
@@ -10,66 +10,144 @@
+  <p>Our Utility Grants program supports developers creating essential utilities, libraries, and tooling. Grants of $5,000-25,000 per project aim to fill critical tooling gaps and strengthen the foundations of an open and interoperable web.</p>
+  <p>For the Spring 2025 cycle, we seek proposals from qualified individuals or teams for the following:</p>
+
+  <h3>RFP #2025-01: CAR Archive Explorers and Utilities</h3>
+  <p>The CAR format (Content Addressable aRchives) makes data reliably portable by packaging up content with its cryptographic identifiers. This ensures the same data can be verified and used across different systems and platforms, making it a good building block for “credible exit” and data portability. Typically found in a file with a <code>.car</code> filename extension, you can think of them like TAR files that are designed for storing collections of content addressed data.</p>
+  <p>Originally created as a way to create stable documents from large or dynamic IPLD graphs (including those graphs that populate “virtual directories” of files added to IPFS), CAR files can now be found in IPFS-based systems, Storacha, ATProto, and other open source projects. With growing interest in data independence – moving large datasets between storage providers, backing up personal content from hosted services, and maintaining data integrity across the web – there's a growing need for accessible tools that help people create, inspect, and manage their own CAR files.</p>
+  <p>Currently, there is limited tooling for the CAR format. Prior art includes Bluesky’s <a href="https://github.com/bluesky-social/indigo/blob/main/cmd/gosky/car.go">car.go</a>, which offers subcommands to work with CAR files on local disk.</p>
+  <p>We are interested in tools that make CAR archives more useful and/or more usable. Command line tools, utility libraries, and browser-based explorers are equally welcome. Enhancements or extensions to existing tools or platforms are especially encouraged.</p>
+
+  <h3>RFP #2025-02: DASL Testing</h3>
+  <p>DASL (Data-Addressed Structures & Links, pronounced "dazzle") is a small set of simple, standard primitives for working with content-addressable resources on today’s or tomorrow’s web. It is a strict subset of IPFS CIDs and IPLD, optimized for simplicity, HTTP, and longevity.</p>
+  <p>The DASL spec was published in December 2024, and is currently supported by <a href="https://dasl.ing/#how">9 implementations or tools</a>. Now, let’s test it. We are interested in proposals with two phases:</p>
+  <ul>
+  <li>First, authoring a set of tests, designing a few happy-path and failure test cases for DASL (specifically its <a href="https://dasl.ing/#spec">CID, dCBOR42, and CAR specs</a>).</li>
```

Comment:
```suggestion
  <li>First, authoring a set of tests and designing a few happy-path and failure test cases for DASL (specifically its <a href="https://dasl.ing/#spec">CID, dCBOR42, and CAR specs</a>).</li>
```
i detect an instance of [my own] comma abuse